.github/renovate.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "automerge": true,
+  "extends": [
+    "config:base",
+    ":semanticCommitsDisabled"
+  ],
+  "reviewers": [
+    "team:badge-plugin-developers"
+  ],
+  "labels": [
+    "dependencies"
+  ],
+  "rebaseWhen": "conflicted",
+  "regexManagers": [
+    {
+      "fileMatch": [
+        ".mvn/extensions.xml"
```

Comment:
This is fixed on recent renovate version (https://github.com/renovatebot/renovate/pull/28893)

I confirm it works for self hosted but not aure about the GitHub app

Reply:
Oh nice, did not know that! Will fix it, thanks for letting me know!